### PR TITLE
Issue template improvements

### PR DESCRIPTION
**Pull request by mrtnptrs**
This template should now be moron/idiot/fool-proof :) Hopefully you will discuss it now, as my previous pull request silently died..... well, I accidentally deleted the old branch with these improvements, so I had to make a new, and much improved, branch. Please discuss it now and try to merge it as soon as possible to make it easier for users to report bugs.

See #3911 for the past discussion.

### CODE REVIEW

**Comment by janisozaur** [comment]
@mrtnptrs please see https://github.com/OpenRCT2/OpenRCT2/wiki/Commit-Messages

There's no reason to include all that drama in commit message.

**Comment by mrtnptrs** [comment]
Do you think this looks good or do the "###" and "####" not really work and should we bring "**xxxxx**" back? Please people, I need imput :)

**Comment by IntelOrca** [comment]
I think its an improvement.

**Comment by Nubbie** [comment]
Would to be honest make it less confusing for people filling it out, maybe we dont need the biggest headers, just normal ones with ### 

Also, one more space between them? ;o 
or dashed lines like

----

```----``

**Comment by mrtnptrs** [comment]
@Nubbie Good point, will look at it!

**Comment by Nubbie** [comment]
Would say ### or nothing it better at all

We shouldn't need the things that makes it confusing to read in the raw format

I would recommend

- Not using ### or 
- Using ### 

Nothing else (hard for people to read stuff with the different formats like ** `````etc )

**Comment by mrtnptrs** [comment]
@Nubbie maybe just:

```
-Operating System:

-Commit/Build: 
[For example: 7b421c8. It can be found at the left bottom corner of the main menu or at the commandprompt window in case of a crash.]

- Reproducible in the original RCT2? (yes or no?):
- Multiplayer affected? (yes or no?):

- Steps to reproduce:
1. 
2. 
3.

- Explanation of the issue:

- Screenshots / Video:
[Drag & drop screenshots here. Use https://vid.me to upload video.]

- Save game:
[Change the file extension to .txt or package to a .zip so that it can be drag & dropped here...]

- Dump file:
[When OpenRCT2 crashed: you can find the dump file at `%userprofile%\documents\OpenRCT2` Look for the latest .DMP file, zip it and drag & drop it here.]

```
Does that look better to you?

**Comment by mrtnptrs** [comment]
Why don't you like that design so much @IntelOrca ? Example: https://github.com/OpenRCT2/OpenRCT2/issues/4130

**Comment by Nubbie** [comment]
Too many uses of ' - ' ¨
Creates balls everywhere

**Comment by IntelOrca** [comment]
@mrtnptrs I prefer your first one.

**Comment by mrtnptrs** [comment]
@IntelOrca But that got a mess with the enters. @Nubbie every "####" got replaced by a "-" which solves the "space/enter issue" and also leaves it clean. But with the "balls" it makes no full use of the space of the window space of course.

**Comment by Nubbie** [comment]
@mrtnptrs I don't like everything to be made into a bullet point, that's the thing :p

**Comment by mrtnptrs** [comment]
```
-Operating System:

-Commit/Build: 
[For example: 7b421c8. It can be found at the left bottom corner of the main menu or at the commandprompt window in case of a crash.]

-Reproducible in the original RCT2? (yes or no?):
-Multiplayer affected? (yes or no?):

-Steps to reproduce:
1. 
2. 
3.

-Explanation of the issue:

-Screenshots / Video:
[Drag & drop screenshots here. Use https://vid.me to upload video.]

-Save game:
[Change the file extension to .txt or package to a .zip so that it can be drag & dropped here...]

-Dump file:
[When OpenRCT2 crashed: you can find the dump file at `%userprofile%\documents\OpenRCT2` Look for the latest .DMP file, zip it and drag & drop it here.]

```

And this? See #4130 .

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
-**Version:** [e.g. 0.0.5]
-**Commit/Build:** [e.g. 426e106]
+#### Reproducible in the original RCT2? (yes or no?):
+#### Multiplayer affected? (yes or no?):
 
-[Explanation of the issue...]
+### Steps to reproduce:
+1. 
+2. 
+3.
```
Comment (Broxzier) [inline .github/ISSUE_TEMPLATE.md]: Trailing space missing

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
-**Screenshots / Video:**
+### Screenshots / Video:
 [Drag & drop screenshots here. Use https://vid.me to upload video.]
 
-**Save game:**
+### Save game:
 [Change the file extension to .txt or package to a .zip so that it can be drag & dropped here...]
+
+### Dump file:
+[When OpenRCT2 crashed: you can find the dump file at `%userprofile%\documents\OpenRCT2` Look for the latest .DMP file, zip it and drag & drop it here.]
```
Comment (janisozaur) [inline .github/ISSUE_TEMPLATE.md]: Breakpad is only enabled on Windows so far, do not assume user has a dump.

**Comment by Broxzier** [comment]
I dislike the big bold text, it's not obvious users have to fill in something there. Keeping it small and simple will make it easier for everyone. The entire screenshot/video/save/dump part can be combined into one, called Attachments or something similar.

**Comment by Broxzier** [comment]
Template proposal:

----

### Technical Information

**Operating System**: _e.g. Windows 7 Home Premium, 32-bit_
**OpenRCT2 Version**: _e.g. 0.0.5_
**Commit/Build**: _e.g. 7b421c8_

### Details

**Reproducible in the original RCT2?**: _yes / no / haven't tested_
**Happens in multiplayer only?**: _yes / no / haven't tested_
**Description**:

... _Explain what happens you did, what happened, and what you'd have expect to happen._

**Steps to reproduce**:

1. ...
2. ...
3. ...

### Attachments

_Attach useful media here by drag & dropping them here. For saved games change the extension **.txt**, or put them in an archive first. Video's can be uploaded to https://vid.me._

**Comment by janisozaur** [comment]
Whenever I file a new issue, I consider it bothersome to remove parts from it, I'd much rather fill in the blanks. I find it much easier to able to just start typing instead of having to select and delete stuff from it, which more often than not invalidates my PRIMARY selection.

**Comment by Broxzier** [comment]
Fair enough, however for someone who wants to submit their very first bug report and has no idea what any of this means it would help. The selecting and removing of text for making issues isn't something you'd be doing very often. It'd only cost you a bit of time, while for new people it would help them, making their reports more useful. It's a bit of a choice we have to make really, making it easier for us, or more clear for others.

**Comment by janisozaur** [comment]
Let me add another bit to that: I have seen default values being untouched in far too many reports to believe they will be if you keep presenting them this way. Or user would only make it ambiguous by putting data into specified fields, but not all and not bothering to strip the only-for-default formatting.

I think the version as of now in this PR would force the user to at least read it and either properly modify it or leave as-is, pre-empting ambiguity.

**Comment by Broxzier** [comment]
I've seen those too, most of those issues were hard to read. They do not provide information at all. I don't know the ratio between users filling them in, and users leaving them there, so you've probably got a better view on what would be better here.

**Comment by Broxzier** [comment]
Apparently you can use use `<!-- HTML comments -->`  to hide things. This can be used to show instructions that won't show up in the issue.

**Comment by Broxzier** [comment]
New proposal with HTML comments
-------------------------------------
Note: Text is wrapped in issue editor
> ```
> 
> 
> ### Technical Information
> 
> 
> **Operating System**: 
> **OpenRCT2 Version**: 
> **Commit/Build**: 
> **32/64-bit OpenRCT2**:
> 
> 
> - [ ] Reproducible in the original RCT2?
> - [ ] Multiplayer
> 
> ### Description
> 
> 
> 
> 
> ### Steps to reproduce
> 
> 1. 
> 2. 
> 3. 
> 
> ### Attachments
> 
> 
> ```


Empty look
-----------
> 
> 
> ### Technical Information
> 
> 
> **Operating System**: 
> **OpenRCT2 Version**: 
> **Commit/Build**: 
> **32/64-bit OpenRCT2**:
> 
> 
> - [ ] Reproducible in the original RCT2?
> - [ ] Multiplayer
> 
> ### Description
> 
> 
> 
> 
> ### Steps to reproduce
> 
> 1. 
> 2. 
> 3. 
> 
> ### Attachments
> 
> 


Filled in look (dummy)
----------------------

> 
> 
> ### Technical Information
> 
> 
> **Operating System**: Windows 10 64-bit
> **OpenRCT2 Version**: 0.0.5
> **Commit/Build**: abcdefg
> **x86/x64 OpenRCT2**: x86
> 
> 
> - [ ] Reproducible in the original RCT2?
> - [X] Multiplayer
> 
> ### Description
> 
> 
> While in multiplayer people can build trees.
> 
> ### Steps to reproduce
> 
> 1. Brush your teeth
> 2. Check the pressure of your tires
> 3. Do groceries
> 4. Play a game of chess
> 
> ### Attachments
> 
> 
> https://wid.me/MySuperAmazingVideoThatDoesNotExist

**Comment by Nubbie** [comment]
@boxier Would remove visible clusters from the posted file - but the problem is often ""confusing"" layouts or something which leads to people filling in the wrong things

Less is more - I presume

Would be sad if we need a site with a bullet proof step-by-step to get a good clean final result

**Comment by Broxzier** [comment]
@Nubbie Even with a minimal template people often don't read, but for those who do these comments will be helpful. They're only displayed for them, and when ignored we don't have to look past them. This template is already quite small, but does provide the things we need. Build info, description, steps to trigger the issue, and attachments. Of course the user can always remove the parts they don't need.

**Comment by IntelOrca** [comment]
I saw that someone had written their description in the HTML comments in an issue earlier which isn't ideal.

**Comment by Broxzier** [comment]
Ah, of course... Then this wouldn't help either.

**Comment by Margen67** [comment]
"If you make something idiot proof, someone will just make a better idiot."

**Comment by Nubbie** [comment]
As the template got a bigger change with Broxiers proposal, we maybe should close it

**Comment by Gymnasiast** [comment]
I don't think this PR is getting anywhere. That's not to say the issue template cannot be improved further, but that would need a fresh look. I'm therefore closing this PR.

**Pull request closed without merging**